ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -132,16 +150,17 @@ const nonEmpty = (item: string): boolean => item && !!item.trim();
 
 const EraRequiredSwitch = (props: {
   tierConfig: InstitutionTierConfig;
-  onChange: (boolean) => void;
+  onToggle: (boolean) => void;
 }) => {
-  const { tierConfig, onChange } = props;
+  const { tierConfig, onToggle } = props;
   const {
     config: { enableRasLoginGovLinking },
   } = useStore(serverConfigStore);
   return (
-    <InputSwitch
-      data-test-id={`${tierConfig.accessTierShortName}-era-required-switch`}
-      onChange={(v) => onChange(v.value)}
+    <CommonToggle
```

Comment:
I like the substitution, but unfortunately something about this breaks the functionality.  It can be switched OFF but not ON.